qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/features/load_features.py

```diff
@@ -0,0 +1,32 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+from mantid import config
+
+
+def load_features():
+    feature_list = {}
+    # this allows for a mangled config
+    try:
+        feature_list=attempt_load()
+    except:
```

Comment:
This except needs to name a specific exception. This will catch stack overflow, out of memory, Python has died a horrible death exception...etc.